ui/src/app/Routes.tsx

```diff
@@ -57,45 +57,34 @@ const Routes = (): JSX.Element => (
         </ErrorBoundary>
       )}
     />
-    {[
-      controllersURLs.controllers.index,
-      controllersURLs.controller.index(null, true),
-    ].map((path) => (
```

Comment:
@huwshimi wasn't there a reason we switched to mapping over an array? Could it be a "preparing for React Router v6" thing?

Reply:
Thats right, in v6 `path` no longer supports an array.

Reply:
Thanks, will revert to using mapping through the array.